mitmproxy/tools/web/app.py

```diff
@@ -475,7 +486,8 @@ def __init__(self, master, debug):
                 FlowContentView),
             (r"/settings", Settings),
             (r"/clear", ClearAll),
-            (r"/options", Options)
+            (r"/options", Options),
+            (r"/options/dump", DumpOptions)
```

Comment:
I followed the pattern of `DumpFlows` befoe, but seems `SaveOptions` sounds more reasonable here.